osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs

```diff
@@ -13,18 +13,13 @@ namespace osu.Game.Screens.Play.HUD
     public class PlayerSettingsOverlay : VisibilityContainer
     {
         private const int fade_duration = 200;
-
         public bool ReplayLoaded;
-
         public readonly PlaybackSettings PlaybackSettings;
-
         public readonly VisualSettings VisualSettings;
-
+        public readonly AudioSettings AudioSettings; 
```

Comment:
space at the line ending

Reply:
ok,thank for this. I wasn't see this space